### DISCUSSION TITLE
ISV Studio closed, Updated process in Microsoft Docs, Partner Center lacks certification option as per 11/06/2024

### DISCUSSION THREAD

**Discussion by deverin** [microsoft/PowerPlatformConnectors#3537] [General]
**UPDATE 21.06.24** ON 18.06 WE HAVE CHECKED OUR PARTNER CENTER AND OPTION IS THERE under Microsoft 365 & Copilot program tab. ISV Studio had connector in certification process when this transfer started. As of now, existing connector was not yet transferred to this new "certification dashboard" under Partner Center

This is now obsolete, and isv studio as domain are not accessible anymore as per 
https://learn.microsoft.com/en-us/power-platform/important-changes-coming#isv-studio-removed

> Once a pull request has been merged to the dev branch, the connector owner can submit the connector for certification through the Connector certification tab in [ISV Studio](https://isvstudio.powerapps.com/). Once certified, the Microsoft Certification team will handle merging the updates from dev to master.

New process was published here
https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-your-package-for-certification

Partner center is still not updated and the option described in the documentation is not available 

#3536

Did anyone here heard any news or updates about the timeline?

**Comment by rvermaquark** [comment]
Hi, I am also unable to use the steps mentioned in the new process as the options mentioned are not showing up. Although I wanted to update the existing certified connector but for that as well the same process is there as certifying a new one. 

I am  Unable to see Microsoft 365 and copilot program under Home>Marketplace offers>Overview, Where as per the new process i will see option to create new offer for power automate connector or manage existing. 
![NoOption](https://github.com/microsoft/PowerPlatformConnectors/assets/144774487/f532144b-51e1-4beb-b1a8-a126509895c7)

**Reply by deverin** [reply]
Make sure that you are part of Microsoft 365 & Copilot program. I see that you do not have the tab for it on this page. It should be there if you are part of this program. You can become part of it under account settings. One you are looking at on the screenshot is Commercial Marketplace program.

As per 18.06 in our partner center Power Platform Connector is available and you should be able to start the certification process.

We already had our connector in process of certification under ISV Studio, but we still cannot see it transferred to this new "certification dashboard"

Here are the steps from documentation, but make sure you are part of Microsoft 365 & Copilot program first.

> 1. Sign in to [Partner Center](https://partner.microsoft.com/dashboard/home).
> 2. Under the Home heading, select Marketplace offers card.
> 3. On the Marketplace offers | Overview page, select Microsoft 365 and Copilot program.
> 4. On the Microsoft 365 and Copilot page, select New offer > Power Platform Connector.
> 5. In the Name field in the New Power Platform Connector screen, enter a unique name for your offer.

**Reply by rvermaquark** [reply]
Hi, thanks for the response. I did enroll into the Microsoft 365 & Copilot program , but it landed me in the (Account settings | Identifiers) page.
Do I need to add a new publisher and add Microsoft 365 & Copilot as a program in that? Such that it shows up in the active programs column? After this will i be able to see the Microsoft 365 & Copilot as registered program under Account settings | Programs page?

**Reply by hbulens** [reply]
^^^ In an identical situation here.
